AngularPart1/Components/AddTodo/addTodo.html

```diff
@@ -0,0 +1,7 @@
+<div ng-controller="addTodoController">
+    <h2>Add a Task</h2>
+    <form>
+        <input type="text" placeholder="" ng-model="newTask.name" required/>
+        <input type="button" value="Add" ng-click="addTask()"/>
```

Comment:
Couldn't find in your code: " Create a custom validator that check the minimum length of a todo text: 20 symbols."